packages/web/package.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "i-just-crashed-my-plane",
+  "version": "0.1.0",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "test:unit": "vue-cli-service test:unit",
+    "test:e2e": "vue-cli-service test:e2e",
+    "lint": "vue-cli-service lint",
+    "docs:generate": "npx typedoc --out docs src",
+    "ci:e2e:start-server": "yarn serve",
+    "ci:e2e:run": "cypress run --record --key 4985f51e-046b-4590-b6e3-f124c50cc39c",
+    "ci:e2e": "npx start-server-and-test ci:e2e:start-server 8080 ci:e2e:run",
+    "ci:unit": "yarn run test:unit"
+  },
+  "dependencies": {
+    "@bugsnag/js": "^7.1.0",
+    "@bugsnag/plugin-vue": "^7.1.0",
+    "cldr-data": "^36.0.0",
+    "cldrjs": "^0.5.1",
+    "core-js": "^3.6.5",
+    "i-just-crashed-my-plane-survey": "^0.1.0",
```

Comment:
Not sure why its not diffing correctly but the difference here is the new dependency on survey package